### PR TITLE
v18.0

### PR DESCRIPTION
**Pull request by a3s7p**
- install wireguard via plan
- fix typo
- wireguard-addclient: ensure allowed_ips is optional
- update changelog

### CODE REVIEW

File: plan/main [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 lighttpd
 lighttpd-mod-openssl
+
+wireguard
```
Comment (JedMeister) [inline plan/main:6]: We should also install explicitly `wireguard-tools` - which should resolve https://github.com/turnkeylinux/tracker/issues/1824

**Review by JedMeister — COMMENTED** [review]
Beyond my note, looks good man.

Only other thing is that it would be really good to handle launch from the Hub better - i.e. resolve https://github.com/turnkeylinux/tracker/issues/1646

I'm pretty sure that the crux of that issue is that the Wireguard inithooks use `APP_IP`, but the Hub doesn't provide that and it should probably be using `APP_VIRTUAL_SUBNET` like OpenVPN does.

I'm not sure, but I think that that should make it "just work", although there is a possibility that something in the Hub also needs a tweak?

**Comment by JedMeister** [comment]
Also, after reviewing OpenVPN, I wonder if we should be setting the timezone in wireguard too (as we do in OpenVPN: https://github.com/turnkeylinux-apps/openvpn/blob/master/overlay/usr/lib/inithooks/firstboot.d/35timezone)?

If so, then perhaps that script should be moved to a common overlay?

**Comment by a3s7p** [comment]
https://github.com/turnkeylinux/common/pull/293

**Comment by JedMeister** [comment]
Awesome, thanks man.

**Pull request merged**